### PR TITLE
Feature/agency location access token retrieval

### PR DESCRIPTION
**Pull request by juanzenn**
@pedrouzcategui 

This pull request is to retrieve all the content you've made here. 😁

### CODE REVIEW

**Review by pedrouzcategui — APPROVED** [review]

File: apis/AuthBuddy/AuthBuddyAPI.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        companyId,
+        name,
+        User: {
+          connect: {
+            id: userId,
+          },
+        },
+      },
+    });
+    return company as Company;
```
Comment (AlfredoPrograma) [inline apis/AuthBuddy/AuthBuddyAPI.ts:22]: Why is this assertion needed? Doesn't `db.company.create` return a `Company`?

File: apis/LeadConnector/LeadConnectorAPI.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        Version: VERSION_DATE,
+      },
+    });
+
+    return location;
+  },
+
+  getLocations: function (companyId: string): Location[] {
+    api.get("/");
+    return [];
```
Comment (AlfredoPrograma) [inline apis/LeadConnector/LeadConnectorAPI.ts:34]: Is this work in progress? I think so

File: apis/LeadConnector/LeadConnectorAPI.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        Version: VERSION_DATE,
+        Accept: "application/json",
+      },
+    });
+
+    return company;
+  },
+
+  createCompany: async function (): Promise<Company> {
+    return {} as Company;
```
Comment (AlfredoPrograma) [inline apis/LeadConnector/LeadConnectorAPI.ts:59]: Is this work in progress? I think so

File: apis/LeadConnector/types/AuthorizationResponse.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+export interface AuthorizationResponse {
```
Comment (AlfredoPrograma) [inline apis/LeadConnector/types/AuthorizationResponse.ts:1]: Could be nit picky but **eslint** should forces you to use `type` instead `interface` :thinking: in order to preserve consistency through codebase.

File: prisma/schema.prisma [hunk 5]
```diff
@@ -47,25 +47,26 @@ model User {
   password            String?
   accounts            Account[]
   sessions            Session[]
-  agencies            Agency[]
+  agencies            Company[]
```
Comment (AlfredoPrograma) [inline prisma/schema.prisma:50]: Could be nit picky; but why don't preserve just one naming? **agencies -> Agency[]** or **companies -> Company[]**.

I mean, consistent naming guarantee better maintenance for codebase in long term. If **agency === company**; why name them different?

**Review by AlfredoPrograma — COMMENTED** [review]
I think @pedrouzcategui  has not configured `prettier` and `eslint` yet. There are some bypassed rules that should be warned or fixed during development :thinking:

**Pull request merged**